overview.html

```diff
@@ -219,6 +220,44 @@ <h3 id="Bulk document CRU">Bulk document CRU</h3>
     </LI>
 </UL>
 
+<h3 id="Document revisions">Document revisions</h3>
+<P>
+    This library does not modify the Java objects passed to the
+    {@link com.cloudant.client.api.Database#save(Object)},
+    {@link com.cloudant.client.api.Database#post(Object)},
+    {@link com.cloudant.client.api.Database#update(Object)},
+    or {@link com.cloudant.client.api.Database#bulk(List)} methods. As such once the server write
+    completes the Java object(s) will be out of sync with the database server regarding the
+    document revision. For any of these methods the new document revision can be obtained from
+    the server's {@link com.cloudant.client.api.model.Response} as shown here.
+</P>
+<pre>
+    {@code
+    Foo foo = new Foo();
+    Response response = db.save(foo);
+    foo.setRevision(response.getRevision());
+    }
+</pre>
```

Comment:
Does this work for status codes other than 2XX -- I guess they throw an exception so you don't get to line 3?